warcprox/mitmproxy.py

```diff
@@ -361,12 +366,15 @@ def do_COMMAND(self):
             self.logger.error("exception proxying request", exc_info=True)
             raise
 
-    def _proxy_request(self):
+    def _proxy_request(self, timestamp=None):
```

Comment:
I'm thinking we should generalize this a bit and make the parameter `extra_response_headers` (a dictionary)

Reply:
Yes, I changed it to ``extra_response_headers`` dict.